src/renderer/components/watch-video-description/watch-video-description.scss

```diff
@@ -9,3 +9,19 @@
   white-space: pre-wrap;
   overflow-wrap: anywhere;
 }
+
+.transcripts {
+
+  .title {
+    font-size: 1.6rem;
+    line-height: 2rem;
+    max-block-size: 2rem;
+    font-weight: 500;
+    color: var(--primary-text-color);
+  }
+
+  .subtext {
+    color: var(--secondary-text-color);
+  }
+
+}
```

Comment:
Not a fan of unnecessary nesting, please pick better class names like `transcriptsTitle` and remove the nesting. Also we use Vue's scoped CSS feature, so Vue already makes sure that the CSS only applies to instances of this component or in some cases it's children, so that's another reason why you should avoid adding unnecessary bloat with nesting.